open_ce/__init__.py

```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 # *****************************************************************
 """
-__version__ = "1.1.3"
+__version__ = "2.0.0"
```

Comment:
Should it be 1.2.0 or 2.0.0? This corresponds to the open-ce release.

Reply:
The build tools will be on a different release cadence than Open-CE. The next release of the build tools will be 2.0, whereas the next release of Open-CE will be 1.2.